video_xblock/static/html/youtube.html

```diff
@@ -0,0 +1,11 @@
+<video
+  id="{{ video_player_id }}"
+  class="video-js vjs-default-skin"
+  controls
+  {% if autoplay %}
+      autoplay
+  {% endif %}
+  width="829" height="511"
```

Comment:
Video player auto-sizing is planned for later iteration.
For now we display fixed sized players.